tensorboard/__init__.py

```diff
@@ -66,6 +66,12 @@
 # additional discussion.
 
 
+@lazy.lazy_load('tensorboard.errors')
```

Comment:
Any reason to make this lazy?  It seems fine to just do `from tensorboard import errors` here right, since that has no dependencies?

Reply:
Aside from just following the existing pattern, I figured that this was
slightly safer in case we introduced a dependency on `tf.errors` down
the line (even via `tb.compat`), though it’s true that the notf smoke
tests added in this commit would catch that. Can change to a direct
import if you prefer.

Reply:
Yeah I think I'd prefer a direct import, just because the lazy loading is a bit harder to reason about and I'd rather not end up having an entire lazy-loaded API namespace just out of an overabundance of caution.

FWIW, assuming we do a hard import here then introducing a non-lazy TF dependency should be caught by the summary dep test as well:
https://github.com/tensorflow/tensorboard/blob/690f4f3c4d102a53f2c4ed1e1854b735ee0c560f/tensorboard/summary/summary_test.py#L110

Reply:
Cool, will do.